### PR TITLE
fix: Velero server args organization

### PR DESCRIPTION
**Pull request by mateusoliveira43**
## Why the changes were made

I think it is easier to read the code if all YAML definitions logic are in `api/v1alpha1` folder.

## How to test the changes made

Check that output of `make bundle` was not changed by these changes.

### CODE REVIEW

**Comment by shubham-pampattiwar** [comment]
cc @kaovilai

File: api/v1alpha1/oadp_types.go [hunk 1]
```diff
@@ -68,6 +67,191 @@ const TechPreviewAck UnsupportedImageKey = "tech-preview-ack"
...
+	// Features  []VeleroFeatureFlag `json:"features,omitempty"`
+
+	// Show colored output in TTY
+	// +optional
+	Colorized *bool `json:"colorized,omitempty"`
+	// Boolean flags. Not handled atomically because the flag.Value interface
+	// does not let us avoid the =true, and that shorthand is necessary for
+	// compatibility. TODO: does this matter enough to fix? Seems unlikely.
+	// +optional
+	ToStderr *bool `json:"logtostderr,omitempty"` // The -logtostderr flag.
```
Comment (kaovilai) [inline api/v1alpha1/oadp_types.go:174]: We are losing context where things came from.

Like that this came from https://github.com/openshift/velero/blob/240b4e666fe15ef98defa2b51483fe87ac9996fb/pkg/cmd/velero/velero.go#L125

Prior code had these in comments. I would add these comments back in.

Unless this refactor is meant to be short lived, I am in favor of having a map of where things came from.

Comment (mateusoliveira43) [reply api/v1alpha1/oadp_types.go:181]: Added back info

**Review by kaovilai — APPROVED** [review]

**Comment by kaovilai** [comment]
timeout

/retest

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**